### PR TITLE
DROTH-291 Batch run to update stops that are missing in the road register

### PR DESCRIPTION
**Pull request by marquesfm**
- Create Batch for verify if Bus Stop existed at OTH exist too at Tierekisteri, if doesn't exist send a request to Tierekisteri for create the Bus Stop.

### CODE REVIEW

File: digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/DataFixture.scala [hunk 1]
```diff
@@ -479,6 +500,61 @@ object DataFixture {
...
+
+    municipalities.foreach { municipality =>
+      println("Start processing municipality %d".format(municipality))
+
+      //Get all OTH Bus Stops By Municipality
+      persistedStop = massTransitStopService.getByMunicipality(municipality, true)
+
+      persistedStop.foreach { stop =>
+        // Validate if OTH stop are known in Tierekisteri and if is maintained by ELY
+        if (!liviIdsListTR.exists(_ == "OTHJ%d".format(stop.nationalId)) && massTransitStopService.isStoredInTierekisteri(Some(stop))) {
```
Comment (tvenhola) [inline digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/DataFixture.scala]: Not all stops are of that form - some stops have older LiviIDs which are saved in stops' properties.

**Comment by halied** [comment]
Should we have separate batch processes for the listing and the update?

**Comment by tvenhola** [comment]
Maybe some kind of dry-run option would be good for trying it out against the server data.

**Comment by tvenhola** [comment]
Looks good to me

**Pull request merged**